Education-analysis.sql

```diff
@@ -0,0 +1,292 @@
+/*1)	Gross intake ratio in first grade of primary education, female (% of relevant age group)*/
+create temp table Gross_intake_1grade_female
+as
+select 	distinct i.countryname
```

Comment:
Dobrze, że kolejne zmienne pojawiają się w nowych wierszach, jednak dobrze byłoby, żeby były do siebie wyrównane. Także odstęp między SELECT a DISTINCT jest niepotrzebny. Przecinki przy wybieraniu wielu zmiennych z kolei umieszczamy po każdej zmiennej. Zalecałbym taką konstrukcję:

```suggestion
select distinct 
                      i.countryname,
                      i."Year" 
                      i.value
```
itd. lub alternatywnie

```suggestion
select distinct 
             i.countryname,
             i."Year" 
             i.value
```
Gdzie wyraźnie masz zaznaczony odstęp między słowem kluczowym DISTINCT, a występującą niżej "wyliczanką" nazwy zmiennych. Obydwie konwencje są ok, ważne, żeby trzymać się ich potem w dalszej części skryptu.

Reply:
Poprawione